blockdag/dag.go

```diff
@@ -1278,6 +1278,13 @@ func (dag *BlockDAG) isCurrent() bool {
 	return dagTimestamp >= minus24Hours
 }
 
+// AdjustedTime returns the adjusted time according to
+// dag.timeSource. See MedianTimeSource.AdjustedTime for
+// more details.
+func (dag *BlockDAG) AdjustedTime() time.Time {
+	return dag.timeSource.AdjustedTime()
```

Comment:
Find all the places where we call `dag.timeSource.AdjustedTime`, and replace with call to `dag.AdjustedTime()`